docs/releasenotes/archive/service/year2018.md

```diff
@@ -5,4 +5,816 @@ sidebar_label: 2018 Archive
+
+* The [Sumo Logic App for Google Cloud Storage](https://help.sumologic.com/07Sumo-Logic-Apps/06Google/Google_Cloud_Storage) helps you monitor activity in Google Cloud Storage. The preconfigured dashboards provide insight into request locations, bucket and object operations, user activities, errors, and bucket statistics.
+
+* The [Sumo Logic App for Google Cloud Functions](https://help.sumologic.com/07Sumo-Logic-Apps/06Google/Google_Cloud_Functions) helps you monitor your usage of Google Cloud Functions. The preconfigured dashboards provide insight into function executions, operations, latency, errors, and failures.
+
+* The [Sumo Logic App for Google Compute Engine](https://help.sumologic.com/07Sumo-Logic-Apps/06Google/Google_Compute_Engine) helps you monitor your infrastructure by providing preconfigured dashboards that allow you to view the activities, users, message severity of your Google Compute Engine infrastructure.
+
+* The [Sumo Logic App for Google Cloud SQL](https://help.sumologic.com/07Sumo-Logic-Apps/06Google/Google_Cloud_SQL) helps you monitor your usage of Google Cloud SQL. The preconfigured dashboards provide insight into created and deleted resources, messages, authorization failures, user activities, and error logs.
+
+---
+## March 27, 2018 (Autocomplete Dashboard Filters)
+
+New Feature - Autocomplete for dashboard filters is now here. Select your Sumo Logic metadata or parsed field from the filters list and start typing in the field you want. Autocomplete will suggest available options in an alphabetized list.
+
+---
+## March 27, 2018 (SAML Required)
+
+New Feature - SAML Required Option. As an administrator, you can now prevent users from accessing Sumo Logic with a password by [requiring SAML Sign In](https://help.sumologic.com/Release-Notes/Service-Release-Notes) for all but a few whitelisted users from the SAML page, or none if you so choose.
+
+--
+## March 27, 2018 (TLS 1.2 Requirement Reminder for Collectors)
+
+Deprecation -  As of May 1, 2018, to keep up with industry standard best practices and ensure the security of our customer data, we will only support TLS version 1.2. All Sumo collector agents must be to upgraded to version 19.155-X or higher while running JRE 7+.
+
+--
+## March 27, 2018 (Search)
+
+New Operator -** **The [topk](https://help.sumologic.com/05Search/Search-Query-Language/Search-Operators/topk) operator is an improved top operator that allows you to select the top values from fields and group them by fields. It can replace the top operator and adds the ability to choose the top of top. If you are using top, we recommend switching to topk for all your queries so that you can take advantage of the additional functionality of topk.
```

Comment:
```suggestion
New Operator - The [topk](https://help.sumologic.com/05Search/Search-Query-Language/Search-Operators/topk) operator is an improved top operator that allows you to select the top values from fields and group them by fields. It can replace the top operator and adds the ability to choose the top of top. If you are using top, we recommend switching to topk for all your queries so that you can take advantage of the additional functionality of topk.
```
```suggestion
New Operator -** **The [topk](https://help.sumologic.com/05Search/Search-Query-Language/Search-Operators/topk) operator is an improved top operator that allows you to select the top values from fields and group them by fields. It can replace the top operator and adds the ability to choose the top of top. If you are using top, we recommend switching to topk for all your queries so that you can take advantage of the additional functionality of topk.
```